autolog-ui/src/contexts/inventory.jsx

```diff
@@ -112,12 +117,16 @@ export const InventoryContextProvider = ({ children }) => {
 
   // Add members to an inventory
   const addInventoryMembers = async (userEmail, InventoryId) => {
+    //TODO: accept user role when backend uses it
+
     const { data, error } = await apiClient.addInventoryMember(userEmail, InventoryId);
+
     if (!error) {
       console.log("Added member is", data);
```

Comment:
comment out please

Reply:
Whoops, already fixed. Thank you for the catch